examples/webgl_animation_skinning_ik.html

```diff
@@ -0,0 +1,225 @@
+				if ( n.name === 'lowerarm_l' ) OOI.lowerarm_l = n;
+				if ( n.name === 'Upperarm_l' ) OOI.Upperarm_l = n;
+				if ( n.name === 'hand_l' ) OOI.hand_l = n;
+				if ( n.name === 'target_hand_l' ) OOI.target_hand_l = n;
+
+				if ( n.name === 'boule' ) OOI.sphere = n;
+				if ( n.name === 'Kira_Shirt' ) OOI.kira = n;
+
+				if ( n.isMesh ) n.frustumCulled = false;
+
+			} );
+			scene.add( gltf.scene );
+
+			orbitControls.target.copy( OOI.sphere.position ); // orbit controls lookAt the sphere
+			OOI.hand_l.attach( OOI.sphere );
+
+			// mirror sphere cube-camera
+			const cubeRenderTarget = new THREE.WebGLCubeRenderTarget( 1024 );
+			mirrorSphereCamera = new THREE.CubeCamera( .05, 50, cubeRenderTarget );
+			scene.add( mirrorSphereCamera );
+			const mirrorSphereMaterial = new THREE.MeshBasicMaterial( { envMap: cubeRenderTarget.texture } );
+			OOI.sphere.material = mirrorSphereMaterial;
+
+			transformControls = new TransformControls( camera, renderer.domElement );
+			transformControls.size = .75;
+			transformControls.space = 'world';
+			transformControls.attach( OOI.target_hand_l );
+			scene.add( transformControls );
+			// disable orbitControls while using transformControls
+			transformControls.addEventListener( 'mouseDown', () => orbitControls.enabled = false );
+			transformControls.addEventListener( 'mouseUp', () => orbitControls.enabled = true );
+
+			OOI.kira.add( OOI.kira.skeleton.bones[ 0 ] );
+			const iks = [
+				{
+					target: 25, // "target_hand_l"
+					effector: 9, // "hand_l"
+					links: [
+
+						{
+							index: 7, // "Upperarm_l"
+							rotationMin: new THREE.Vector3( 0.1, - 0.7, - 1.8 ),
+							rotationMax: new THREE.Vector3( 1.1, 0, - 1.4 )
+						},
+						{
+							index: 8, // "lowerarm_l"
+							rotationMin: new THREE.Vector3( 1.2, - 1.8, - .4 ),
+							rotationMax: new THREE.Vector3( 1.7, - 1.1, .3 )
+						},
+						{
+							index: 9, // "hand_l"
+							rotationMin: new THREE.Vector3( 0, - .5, 0.3 ),
+							rotationMax: new THREE.Vector3( 0, - .5, .3 )
+						},
+					],
+				}
+			];
+			IKSolver = new CCDIKSolver( OOI.kira, iks );
+			const ccdikhelper = new CCDIKHelper( OOI.kira, iks, { sphereSize: .01 } );
+			scene.add( ccdikhelper );
+
+			gui = new GUI();
+			gui.add( conf, 'followSphere' ).name( 'follow sphere' );
+			gui.add( conf, 'turnHead' ).name( 'turn head' );
+			gui.add( conf, 'ik_solver' ).name( 'IK auto update' );
+			gui.add( IKSolver, 'update' ).name( 'IK manual update()' );
+			gui.open();
+
+			window.addEventListener( 'resize', onWindowResize, false );
+
+		}
+
+		function animate( ) {
+
+			if ( OOI.sphere && mirrorSphereCamera ) {
+
+				OOI.sphere.visible = false;
+				OOI.sphere.getWorldPosition( mirrorSphereCamera.position );
+				mirrorSphereCamera.update( renderer, scene );
+				OOI.sphere.visible = true;
+
+			}
+
+			if ( OOI.sphere && conf.followSphere ) {
+
+				// orbitControls follows the sphere
+				OOI.sphere.getWorldPosition( v0 );
```

Comment:
FYI: When using `getWorldPosition()` inside `animate()`, it's good for performance to avoid creating an instance of `Vector3` on each invocation. Instead the code uses now existing objects.